src/main/java/com/powsybl/openloadflow/sensi/DcSensitivityAnalysis.java

```diff
@@ -269,8 +269,13 @@ protected void calculateSensitivityValues(List<LfSensitivityFactor<DcVariableTyp
         if (lfFactors.isEmpty()) {
             return;
         }
-        lfFactors.stream().filter(factor -> factor.getStatus() == LfSensitivityFactor.Status.ZERO)
-                .forEach(factor -> valueWriter.write(factor.getIndex(), contingency != null ? contingency.getIndex() : -1, 0, Double.NaN));
+        for (LfSensitivityFactor<DcVariableType, DcEquationType> lfFactor : lfFactors) {
+            if (lfFactor.getStatus() == LfSensitivityFactor.Status.ZERO) {
+                valueWriter.write(lfFactor.getIndex(), contingency != null ? contingency.getIndex() : -1, 0, Double.NaN);
+            } else if (contingency != null && lfFactor.getStatus() == LfSensitivityFactor.Status.SKIP) {
```

Comment:
I don't know yet what is the right fix but not this one because, all skipped factors are handle at the start of DC sensitivity analysis. So here a value is emitted in the writer a second time.